packages/core/package.nls.json

```
@@ -129,6 +129,7 @@
     "AWS.command.ec2.stopInstance": "Stop EC2 Instance",
     "AWS.command.ec2.rebootInstance": "Reboot EC2 Instance",
     "AWS.command.ec2.copyInstanceId": "Copy Instance Id",
+    "AWS.command.ec2.viewLogs": "View Current EC2 System Logs",
```

is the word "current" adding anything useful?
```suggestion
    "AWS.command.ec2.viewLogs": "View EC2 Logs",
```

---

I believe you can setup a CWL group to include all metrics from the ec2 instances, but it doesn't come pre-setup from what I can tell. So the system logs differ from that. 

I included current because the ec2 logs don't have the "load newer", "load older" codelenses, so its really just a snapshot of the current system logs.

---

> I included current because the ec2 logs don't have the "load newer", "load older" codelenses, so its really just a snapshot of the current system logs.

That's a good point, but it may be more confusing to use different language in actions. We can always provide detailed messaging when the logs are actually loaded, but actions themselves should use very consistent language.